### PR TITLE
Closing NZF site 42

### PR DESCRIPTION
**Pull request by EvanCGNS**
Changed the end time and date for NZF (site 42) to 2024-07-02T00:00:00Z UTC (12pm, Tuesday 2nd July NZST).

Applied this change to gauges, recorders, streams, firmware, and sites files.

NOTE: All closing times will be updated to the exact time after deployment (in the re-opening ticket). This pre-deployment merge is to block Tilde data ingestion only.

### CODE REVIEW

**Review by salichon — APPROVED** [review]
legit

**Comment by ozym** [comment]
There's a change to NZG as well in `recorders.csv` which needs clarification.

File: install/recorders.csv [hunk 1]
```diff
@@ -2355,8 +2355,7 @@ SAIC,BPR Subsystem,BPR Subsystem,201004,NZE,41,0,,0,0,2021-12-14T00:00:00Z,2023-
 SAIC,BPR Subsystem,BPR Subsystem,201005,NZA,41,0,,0,0,2021-12-18T00:00:00Z,2023-12-15T00:00:00Z
 SAIC,BPR Subsystem,BPR Subsystem,210325,NZB,41,0,,0,0,2022-08-15T20:15:00Z,2023-12-06T00:00:00Z
 SAIC,BPR Subsystem,BPR Subsystem,220112,NZH,41,0,,0,0,2022-08-01T20:15:00Z,2024-06-23T12:00:00Z
-SAIC,BPR Subsystem,BPR Subsystem,220113,NZF,42,0,,0,0,2022-08-11T00:15:00Z,9999-01-01T00:00:00Z
-SAIC,BPR Subsystem,BPR Subsystem,56951,NZG,42,0,,0,0,2024-06-22T23:30:00Z,9999-01-01T00:00:00Z
```
Comment (ozym) [inline install/recorders.csv:2359]: should this be removed?

**Review by ozym — CHANGES_REQUESTED** [review]
There's a change to NZG as well in `recorders.csv` which needs clarification.

File: install/recorders.csv (hunk 1 above)
Comment (EvanCGNS) [reply install/recorders.csv:2359]: Just fixed it, my mistake

**Review by ozym — APPROVED** [review]
looks good

**Pull request closed without merging**